server/routes/api/graphql.ts

```diff
@@ -40,9 +40,15 @@ const resolvers: IResolver = {
 					$gt: args.pagination_token
 				}
 			} : {};
+			const uuidQuery = args.ids ? {
+				uuid: {
+					$in: args.ids
+				}
+			}: {};
```

Comment:
Ok this is _super_ picky but can you  `--amend` this commit to evenly space around that colon like on line 42?